core-web/libs/portlets/dot-experiments/portlet/src/lib/dot-experiments-configuration/store/dot-experiments-configuration-store.ts

```diff
@@ -973,6 +974,13 @@ export class DotExperimentsConfigurationStore extends ComponentStore<DotExperime
         const configProps = route.snapshot.data['config'];
         const hasEnterpriseLicense = route.parent.snapshot.data['isEnterprise'];
         const pushPublishEnvironments = route.parent.snapshot.data['pushPublishEnvironments'];
+
+        console.info('route', route);
```

Comment:
All from dotcms-ui need to be moved to @ui

Reply:
This will be resolved in another PR. I'll take the work from here.